### PR TITLE
[Staging to Master] 

### PR DESCRIPTION
**Pull request by ksatchit**
- Doc Changes made for 1.1 will be pushed into docusaurus master

### CODE REVIEW

File: docs/openebs-pool-pod-failure.md [hunk 1]
```diff
@@ -136,32 +185,33 @@ metadata:
...
     - name: openebs-pool-pod-failure
       spec:
         components:
-          - name: FORCE
-            value: 'true'
-          - name: APP_PVC
-            value: 'pvc-c466262a-a5f2-4f0f-b594-5daddfc2e29d'    
-          - name: DEPLOY_TYPE
-            value: deployment        
+          env:
```
Comment (ispeakc0de) [inline docs/openebs-pool-pod-failure.md:208]: @ksatchit We have `components.env` field here But this field is not present in our 1.0.0 release. It is only present in ci image. Is it okay to put this change here?

Comment (ksatchit) [reply docs/openebs-pool-pod-failure.md:208]: this is the intention of putting it into master :)  -- this will go into docs master ..so 'next' in URL

Comment (ispeakc0de) [reply docs/openebs-pool-pod-failure.md:208]: okay got it, Thanks

**Review by ispeakc0de — DISMISSED** [review]
/lgtm

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**